src/ui/CategoryFacet/CategoryValue.ts

```diff
@@ -80,14 +80,29 @@ export class CategoryValue implements CategoryValueParent {
     return this.categoryChildrenValueRenderer.children;
   }
 
+  public makeSelectable() {
+    this.label = $$(this.element.find('.coveo-category-facet-value-label'));
+    this.label.addClass('coveo-selectable');
+
+    new AccessibleButton()
+      .withElement(this.label)
+      .withSelectAction(() => this.onSelect())
```

Comment:
How is the keyboard navigation of the value? Should it also have a `withEnterAction` maybe?

Reply:
`withSelectAction()` adds both the keyboard + mouse click